assets/chef-recipes/cookbooks/user_creation/recipes/default.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+user node['user_creation']['name'] do
```

Comment:
Please directly specify the home directory.

Also introduce a variable with that resource.